perun-core/src/test/java/cz/metacentrum/perun/core/entry/ResourcesManagerEntryIntegrationTest.java

```diff
@@ -626,7 +626,7 @@ public void removeGroupFromResourceWhichIsSubGroup() throws Exception {
 		subGroup = setUpSubGroup(group);
 		facility = setUpFacility();
 		resource = setUpResource();
-		resourcesManager.assignGroupToResource(sess, group, resource);
+		resourcesManager.assignGroupToResource(sess, group, resource, false);
```

Comment:
The build is not passing because of line 659, where the same parameter needs to be added.